README.md

```diff
@@ -1,5 +1,60 @@
-# game
+# Design Pattern Island Game
 
-Student/player implementation of the design_patterns_island game!
+## Quick Start
 
-Documentation can be found on [Github (./docs)](./docs), [PyPi](https://pypi.org/project/game/), or [Hosted](https://game.kyleking.me/)!
+### Python
+
+<!-- FIXME: Replace with pyinstaller-based binary -->
+
+### Graphviz
+
+Make sure to [install graphviz from the official website](https://www.graphviz.org/download/)
+
+For Mac users, this might be `brew install graphviz`, while Windows may need to download the latest `.exe` installer from the site or [use chocolatey](https://community.chocolatey.org/packages/graphviz)
+
+## Game Play
+
+> Make sure you go through the installation steps first!
+
+### Quick Start
+
+Let's launch the game UI and make the first code changes!
+
+```sh
+> poetry shell
+(shell) doit play
```

Comment:
I actually see `(game-py3.10)` instead of `(shell)`. That was the case for the participants too, except that was back when we were using community rpg so the shell was `(community-rpg-py3.10)`.

Reply:
Good catch!